.gitignore

```diff
@@ -113,3 +113,4 @@ token.txt
 docs/build/
 docs/source/_build
 **/.vscode/
+/src/wechaty/version.py
```

Comment:
We should not ignore it because this belongs to our repo.

Reply:
> BTW: I'm working on https://wechaty.js.org/docs/recipes/events/ and trying to add TS/JS events example code to it.
> 
> Could you please have a look at it, and fill the python language part of those events when you get some time?
> 
> And suggestions are welcome for this Polyglot Wechaty docs page because I'm quite satisfied with it and want to move on with this layout. (Tabs for all programing languages)

OK， I'll take a look later.